classes/local/manager.php

```diff
@@ -29,6 +29,7 @@ public static function set_objectfs_config($config) {
     }
 
     public static function get_objectfs_config() {
+        global $CFG;
```

Comment:
missing php doc above? why didn't ci fail this?

Reply:
updated